spec/rails_helper.rb

```diff
@@ -43,6 +43,7 @@
 
   # You can uncomment this line to turn off ActiveRecord support entirely.
   # config.use_active_record = false
+  config.include Capybara::DSL
 
   # RSpec Rails can automatically mix in different behaviours to your tests
   # based on their file location, for example enabling you to call `get` and
```

Comment:
To answer your reflection question about the reason why `visit` wasn't working in your `hotels/show_spec.rb` file. 

If you remember, we said that if you wanted, you could add this to your feature tests: `type: :feature`, and if you didn't Rails would determine what type of test it was by looking at the folder where the test was located. Well in your `hotels/show_spec.rb` file, you didn't specify the `type`, which is ok, but you also didn't put `hotels/` in the `features/` folder so Rails was unable to determine what type of test this was. `visit` is only available in feature tests.

Moving `hotels/` into `features/` was all you needed. You don't need the `config.include Capybara::DSL` line.